_posts/2017/2017-12-11-set-title-in-the-view-template-using-content-for.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Set your page title in the view template using content_for'
+description: 'Cut out all that instance variable nonsense.'
+layout: article
+category: ruby
+image:
+  base: '2017/set-title-in-the-view-template-using-content-for'
+  alt: 'Books'
+  credit: 'Annie Spratt'
+  source: 'https://unsplash.com/photos/-9vMBjrU-RA'
+---
+
+Rails has `content_for` as it’s primary way to store content in one place for use in other views, layouts or helpers. See `content_for` in the Rails documentation at [in the ActionView helpers](http://api.rubyonrails.org/v5.1.3/classes/ActionView/Helpers/CaptureHelper.html#method-i-content_for).
```

Comment:
* Rails _uses_ `content_for` as _its_ primary way to store content...
(Emphasis for you, not for the post. :-) )

Reply:
*Find out more about `content_for` in the Rails documentation in the [ActionView helpers] section

Reply:
I weep for my own grammar. It's super-weird to me that this doesn't stand out reading back my own writing, but I spot it so easily in other people's stuff.

Reply:
Oh yeah, always easier when you're reading somebody else's stuff.

I like to think I'm very good at this but JB always manages to make MASSIVE improvements to anything I write, so...